packages/modules/viewer-react/src/index.ts

```diff
@@ -10,4 +10,4 @@ export * from "./utilities";
 export * from "./types";
 export * from "./hooks";
 export * from "oidc-client";
-export * from "./services/auth/ViewerAuthorizationClient";
+export * from "./services/auth";
```

Comment:
sweet!

making a comment here so we can revisit re-exporting the oidc-client pkg, and possiibly removing this altogether before next release